db/fixtures/miq_product_features.yml

```diff
@@ -2570,20 +2570,12 @@
     :feature_type: view
     :identifier: tasks_view
     :children:
-    - :name: All VM and Container Analysis Tasks
-      :description: Display Lists of All VM and Container Analysis Tasks
-      :feature_type: view
-      :identifier: job_all_smartproxy
```

Comment:
Alright, I see.

IIRC, in the past, we had not created data migrations for feature removals.